tests/IsValidUtf8.hs

```diff
@@ -29,7 +29,7 @@ testSuite = testGroup "UTF-8 validation" $ [
     goInvalid = forAll arbitrary $ 
       \inv -> (B.isValidUtf8 . toByteString $ inv) === False
     testCount :: QuickCheckTests
-    testCount = 100000
+    testCount = 1000
```

Comment:
Are these tests still effective at 1/100 the number of tests?

Maybe @kozross can tell?!

Reply:
I suspect they might be still. The tests are taken straight from the ones in `text`, so if they're effective at 1000, so will these be.